keras/layers/recurrent_convolutional.py

```diff
@@ -0,0 +1,523 @@
+from .. import backend as K
+from .. import activations, initializations, regularizers
+
+import numpy as np
+from ..engine import Layer, InputSpec
+from ..utils.np_utils import conv_output_length
+import warnings
+
+
+class ConvRecurrent2D(Layer):
```

Comment:
If we adopt the naming convention `ConvRecurrent` / `ConvLSTM` then the module should be renamed `convolutional_recurrent` as well.